content/blog/2021-04-30-april-21-community-gems.md

```diff
@@ -0,0 +1,235 @@
+your DVC and CML questions answered!
+
+## DVC
+
+### [What is the best way to commit 2 experiment runs?](https://discord.com/channels/485586884165107732/485596304961962003/836626346544594995)
+
+You want to use `dvc exp branch` if you want to keep multiple experiments. That
+way, each one is in a separate branch rather than trying to apply one experiment
+on top of another.
+
+### [How can I clean up the remote caches after a lot of experiments and branches have been pushed?](https://discord.com/channels/485586884165107732/485596304961962003/831142466169733120)
+
+`dvc exp gc` requires some kind of flags to operate. At the very least,
+`--workspace`. So, with `--workspace`, `dvc` will try to read all of the pointer
+files: _.dvc_ files and _dvc.yaml_ files in the workspace. It will read all of
+them and will determine all the cache objects/files that need to be preserved
+(since they are being used in the current workspace). The rest of the files in
+the _.dvc/cache_ are removed.
+
+_This does not require any Git operations!_
+
+You can also use the `--all-branches` flag. It will read all of the files
+present in the current workspace and from the commits in the branches you have
+locally. Then it will use that list to determine what to keep and what to
+remove.
+
+If you need to read pointer files from given tags you have locally, the
+`--all-tags` flag is the best option.
+
+The `--all-commits` flag reads pointer files from every commit and it will make
+a list of all the files that are in the cache/remote and if the _.dvc_ file
+isn't found in any commits of the Git repo, it will delete those files.
+
+It doesn't check if the directories are empty and you can find info and comment
+on this known issue, head to
+[Issue #3375 here.](https://github.com/iterative/dvc/issues/3375) Your input is
+ALWAYS valuable to us! 🙏🏼
+
+### [If I have two cloud folder links added to the DVC config, I'm able to push the data to the default one. How could I push the data to the other cloud folder?](https://discord.com/channels/485586884165107732/563406153334128681/833176227762274364)
+
+You're looking for the `-r / --remote` option for `dvc push`. The command looks
+like this:
+
+```dvc
+$ dvc push --remote <name_of_remote_storage>
+```
+
+It will push directly to the remote storage you defined in the command above.
+
+### [What's the current recommended way to automate hyperparameter search when using DVC pipelines?](https://discord.com/channels/485586884165107732/563406153334128681/829803720190590986)
+
+Take a look at the new
+[experiments feature](https://dvc.org/doc/start/experiments)! It enables you to
+easily experiment with different parameter values.
+
+You could script a grid search pretty easily by queueing an experiment for each
+set of parameter values you want to try. For example:
+
+```dvc
+$ dvc exp run --queue -S alpha={alpha},beta={beta}
+$ dvc exp run --run-all --jobs 2
```

Comment:
should probably mention what does `--jobs 2` do in this case (and this just to show an example, and optional?)

Reply:
Yep! This is just to show an example.